Manifests/ManagedPreferencesApplications/edu.ncsu.confboard.plist

```diff
@@ -0,0 +1,285 @@
+			<string>PayloadIdentifier</string>
+			<key>pfm_require</key>
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>Payload Identifier</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>edu.ncsu.confboard</string>
+			<key>pfm_description</key>
+			<string>The type of the payload, a reverse dns string.</string>
+			<key>pfm_description_reference</key>
+			<string>The payload type.</string>
+			<key>pfm_name</key>
+			<string>PayloadType</string>
+			<key>pfm_require</key>
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>Payload Type</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Unique identifier for the payload (format 01234567-89AB-CDEF-0123-456789ABCDEF)</string>
+			<key>pfm_description_reference</key>
+			<string>A globally unique identifier for the payload. The actual content is unimportant, but it must be globally unique. In macOS, you can use uuidgen to generate reasonable UUIDs.</string>
+			<key>pfm_format</key>
+			<string>^[0-9A-Za-z]{8}-[0-9A-Za-z]{4}-[0-9A-Za-z]{4}-[0-9A-Za-z]{4}-[0-9A-Za-z]{12}$</string>
+			<key>pfm_name</key>
+			<string>PayloadUUID</string>
+			<key>pfm_require</key>
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>Payload UUID</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<integer>1</integer>
+			<key>pfm_description</key>
+			<string>The version of the whole configuration profile.</string>
+			<key>pfm_description_reference</key>
+			<string>The version number of the individual payload.
+A profile can consist of payloads with different version numbers. For example, changes to the VPN software in iOS might introduce a new payload version to support additional features, but Mail payload versions would not necessarily change in the same release.</string>
+			<key>pfm_name</key>
+			<string>PayloadVersion</string>
+			<key>pfm_require</key>
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>Payload Version</string>
+			<key>pfm_type</key>
+			<string>integer</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>This value describes the issuing organization of the profile, as displayed to the user</string>
+			<key>pfm_name</key>
+			<string>PayloadOrganization</string>
+			<key>pfm_title</key>
+			<string>Payload Organization</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>https://storage.googleapis.com/confboard</string>
+			<key>pfm_description</key>
+			<string>The base URL to the folder containing the images.txt file or JSON information. Do NOT include trailing backslash.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://go.ncsu.edu/confboard#h.b53vj55iw28d</string>
+			<key>pfm_format</key>
+			<string>^https://.*$</string>
```

Comment:
Update the regex here to `^https://.*[^/]$` to prevent admins from including a trailing slash as you list in description.
![Screen Shot 2022-11-13 at 6 50 52 PM](https://user-images.githubusercontent.com/11789931/201551449-d9a567bf-f97d-4b5c-8239-5193b62bb864.png)
![Screen Shot 2022-11-13 at 6 50 59 PM](https://user-images.githubusercontent.com/11789931/201551456-bc394305-190d-479b-b716-46b88f53a78a.png)

Reply:
Thank you for the review and comments.  I revised the pfm_format key per your suggestion.  Additionally, I removed pfm_require in the custom keys to allow for all to be optional.